INSTALL.md

```diff
@@ -50,38 +50,14 @@ root filesystem.
 
 *-- Ken*
 
-### 2. Mac OS X Installation
-
-Installing PCP on MacOSX is done via https://brew.sh/ commands.
-From a Terminal run:
-```
-$ brew install qt
-$ brew link qt --force
-$ brew install pcp
-$ brew link pcp
-$ pcp --version
-```
-
-The output for the last command will be something like
-```
-pcp version 4.1.1
-```
-
-Use the version number for creating symlinks (for version 4.1.1)
-```
-$ export version="4.1.1"
-$ sudo ln -s /usr/local/Cellar/pcp/$version/etc/pcp.conf /etc/pcp.conf
-$ sudo ln -s /usr/local/Cellar/pcp/$version/etc/pcp.env /etc/pcp.env
-```
-
```

Comment:
Instead of simply deleting the MacOS section, please replace it with wording like for AIX below and add the URLs to the packaging issues (the brew removal issue and the conflicting package name issue).  This will provide more context for the next person who wishes to work on this.  Thanks!